test/unit/ensure_not_used_by_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'test_helper'
+
+class EnsureNotUsedByTest < ActiveSupport::TestCase
+  def setup
+    @org1 = FactoryGirl.create(:organization)
+    @org2 = FactoryGirl.create(:organization)
+    @user = User.create :auth_source => auth_sources(:one), :login => "foo", :mail  => "foo@bar.com"
+    @user.organizations = [@org1]
+  end
+
+  test "#hostgroup_should_not_be_deleted_if_used_by_host_in_user_org" do
```

Comment:
Nitpick, when using `test` you can use spaces in the description - use underscores if defining a test_* method by hand.  The "#" at the beginning should be removed, it really means an instance method (e.g. testing Host#foo).